src/vmm/src/lib.rs

```diff
@@ -905,8 +905,8 @@ impl Vmm {
     fn start_vcpus(
         &mut self,
         mut vcpus: Vec<Vcpu>,
-        vmm_seccomp_filter: SeccompFilter,
-        vcpu_seccomp_filter: SeccompFilter,
+        vmm_seccomp_filter: BpfProgram,
+        vcpu_seccomp_filter: BpfProgram,
```

Comment:
Since `BpfProgram`s are read-only, maybe these could be `&BpfProgram`.

Reply:
Yep, this is what clippy complains about too (well technically it complains about the double ptr, but I'm still turning them into slices).